r2/r2/lib/mr_top.py

```diff
@@ -134,6 +138,10 @@ def store_keys(key, maxes):
                    for item in maxes]
     query._replace(item_tuples)
 
+    if category == "sr" and thing_cls == "comment":
+        assert query2
+        query._replace(item_tuples)
```

Comment:
It might be better to just leave /r/all's `get_all_comments` method to run on demand instead of being calculated in this way.